pkg/model/backup_routine.go

```diff
@@ -5,24 +5,35 @@ import (
 )
 
 const (
-	MIN_FULL_BACKUP_INTERVAL_MILLIS int64 = 10000
-	MIN_INCR_BACKUP_INTERVAL_MILLIS int64 = 1000
+	minimumFullBackupIntervalMillis int64 = 10000
+	minimumIncrBackupIntervalMillis int64 = 1000
 )
 
 // BackupRoutine represents a scheduled backup operation routine.
+// @Description BackupRoutine represents a scheduled backup operation routine.
 type BackupRoutine struct {
-	BackupPolicy  string  `yaml:"backup-policy,omitempty" json:"backup-policy,omitempty"`
-	SourceCluster string  `yaml:"source-cluster,omitempty" json:"source-cluster,omitempty"`
-	Storage       string  `yaml:"storage,omitempty" json:"storage,omitempty"`
-	SecretAgent   *string `yaml:"secret-agent,omitempty" json:"secret-agent,omitempty"`
+	// The name of the corresponding bakup policy.
+	BackupPolicy string `yaml:"backup-policy,omitempty" json:"backup-policy,omitempty"`
+	// The name of the corresponding source cluster.
+	SourceCluster string `yaml:"source-cluster,omitempty" json:"source-cluster,omitempty"`
+	// The name of the corresponding storage provider configuration.
+	Storage string `yaml:"storage,omitempty" json:"storage,omitempty"`
+	// The Secret Agent configuration for the routine (optional).
+	SecretAgent *string `yaml:"secret-agent,omitempty" json:"secret-agent,omitempty"`
 
-	IntervalMillis     *int64 `yaml:"interval,omitempty" json:"interval,omitempty"`
+	// The interval for full backup in milliseconds.
+	IntervalMillis *int64 `yaml:"interval,omitempty" json:"interval,omitempty"`
+	// The interval for incremental backup in milliseconds.
 	IncrIntervalMillis *int64 `yaml:"incr-interval,omitempty" json:"incr-interval,omitempty"`
 
-	Namespace string   `yaml:"namespace,omitempty" json:"namespace,omitempty"`
-	SetList   []string `yaml:"set-list,omitempty" json:"set-list,omitempty"`
-	BinList   []string `yaml:"bin-list,omitempty" json:"bin-list,omitempty"`
-	NodeList  []Node   `yaml:"node-list,omitempty" json:"node-list,omitempty"`
+	// The name of the namespace to back up.
+	Namespace string `yaml:"namespace,omitempty" json:"namespace,omitempty"`
+	// The list of backup set names.
```

Comment:
// The list of backup set names (optional, an empty list implies backing up all sets.).